docs/to_clarify.md

```diff
@@ -6,7 +6,7 @@
 
 1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
 
-1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems?
+1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems? If there is documentation of these systems, it would be easier for integration.
```

Comment:
I think what you added here isn't meaningful because, duh, documentation definitely helps in integration.

What we should be adding is: 

> If so, what are the APIs used to communicate between these systems? **Will documentation & assistance be provided if we face certain issues interfacing with these systems?**

This sentence should also be merged with the above point

> What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?

Reply:
I dont see anything to add anymore. It is concise enough.

Reply:
I think that we should ask whether they will provide assistance to create additional API for us in the event what we need isn't provided by the existing systems. Or do we have to inherit their code base and implement the API on our own.

Reply:
It will be great if they provide assistance. This would cut time rather than dig from scratch. Good point to note sir!
 will add in.